src/Components/ImagesTable/ImagesTable.js

```diff
@@ -267,7 +267,8 @@ const ImagesTable = () => {
                       </Td>
                       <Td>
                         {compose.request.image_requests[0].upload_request
-                          .type === 'aws' ? (
+                          .type === 'aws' &&
+                        compose?.image_status?.status === 'success' ? (
```

Comment:
This hides the 'Share to region' action. Could we just disable it instead?

I checked the PF source code and I think that if you add an `isDisabled: true` property to the `awsActions` (line 144) it should work.

Reply:
Fixed